rustfmt.toml

```diff
@@ -7,6 +7,9 @@ imports_granularity = "Crate"
 reorder_imports = true
 # Consistency
 newline_style = "Unix"
+# Format comments
+comment_width = 100
+wrap_comments = true
```

Comment:
These options are unstable. I don't know exactly what happens, but I expect stable `rustfmt` to not be too happy.

Reply:
We are already using unstable format options without this.

Reply:
indeed.

Reply:
It's pretty easy to call the nightly rustfmt from stable. Potentially we can push to stabilise the features we're using - quite often things are unstable until someone drops a PR to propose to stabilise them.